### PR TITLE
Update korean.lang

### PR DESCRIPTION
**Pull request by alphaimetal0903**
### Description
Updated version of #4127 
Grammar errors and incorrect translation solutions.

---
**Target Minecraft Versions:** any
**Requirements:** none
**Related Issues:** #4127 , #3999

### CODE REVIEW

**Review by TPGamesNL — CHANGES_REQUESTED** [review]
So I can't read Korean, so with this review I got help from Peter (Discord: Peter_#9686) (thanks again Peter :D)

File: src/main/resources/lang/korean.lang [hunk 1]
```diff
@@ -42,67 +42,67 @@ skript command:
...
+		x scripts in folder: <gold>%1$s<reset> 폴더 속 <gold>%2$s<reset>개의 스크립트
+		empty folder: <gold>%s<reset>에 활성화된 스크립트가 없습니다.
 	enable:
 		all:
-			enabling: 비활성화 된 모든 스크립트를 활성화하는 중...
-			enabled: 이전에 비활성화 된 모든 스크립트를 성공적으로 활성화 및 파싱했습니다.
-			error: 비활성화 된 스크립트를 구문 분석하는 동안 %s개의 오류가 발생했습니다!
-			io error: 스크립트를 로드 할 수 없습니다 (일부 스크립트의 이름이 이미 변경되어 서버가 다시 시작될 때 활성화 됨): %s
+			enabling: 비활성화된 모든 스크립트를 활성화하는 중...
+			enabled: 이전에 비활성화된 모든 스크립트를 성공적으로 활성화하고 구문을 분석했습니다.
```
[the remark this answers is not in the document]
Comment (alphaimetal0903) [reply src/main/resources/lang/korean.lang:70]: I think my translation is correct according to the default language file.

File: src/main/resources/lang/korean.lang [hunk 2]
```diff
@@ -42,67 +42,67 @@ skript command:
...
 		
-	invalid script: 스크립트 폴더에서 <grey>'<gold>%s<grey>'<red>라는 스크립트를 찾을 수 없습니다!
-	invalid folder: 스크립트 폴더에서 <grey>'<gold>%s<grey>'<red>라는 폴더를 찾을 수 없습니다!
+	invalid script: 스크립트 폴더에서 <grey>'<gold>%s<grey>'<red> 스크립트를 찾을 수 없습니다!
+	invalid folder: 스크립트 폴더에서 <grey>'<gold>%s<grey>'<red> 폴더를 찾을 수 없습니다!
 	reload:
 		reloading: %s을(를) 다시 로드하는 중입니다...
 		reloaded: %s을(를) 다시 로드했습니다.
-		error: %1$s을(를) 다시 로드하는 동안 %2$s개의 오류가 발생했습니다!
+		error: %1$s을(를) 다시 로드하는 동안 %2$s개의 오류를 찾았습니다!
```
[the remark this answers is not in the document]
Comment (alphaimetal0903) [reply src/main/resources/lang/korean.lang:56]: _todo_

File: src/main/resources/lang/korean.lang (hunk 1 above)
[the remark this answers is not in the document]
Comment (alphaimetal0903) [reply src/main/resources/lang/korean.lang:70]: Hmm... it's more natural to not have `'이전에'`.

**Review by TPGamesNL — APPROVED** [review]

**Pull request merged**